### DISCUSSION TITLE
Rendering without client-side JS

### DISCUSSION THREAD

**Discussion by omurad** [vuejs/docs#1440] [Q&A]
Is there a built in option in `server-renderer` for rendering without any client-side JS? 
Similar to NuxtJS's `injectScripts` option.

**Comment by tiger-githubb** [comment]
> Is there a built in option in `server-renderer` for rendering without any client-side JS? 
> Similar to NuxtJS's `injectScripts` option.
> 

I think yes it exists

**Comment by tiger-githubb** [comment]
> Is there a built in option in `server-renderer` for rendering without any client-side JS? 
> Similar to NuxtJS's `injectScripts` option.
> 

I think yes it exists
